<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Gateway-PC" ignored="0" name="Test" tests="99" failures="55" timestamp="21 Dec 2019 09:20:22 GMT" time="6.692" errors="0">
  <testcase name="CreateSurpluslocation" time="2.773" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;errorInfo&amp;apos; with value &amp;apos;Success&amp;apos; in response json failed expected [Success] but found [Surplus Location Category Doesnot exist.]">
      <![CDATA[java.lang.AssertionError: Verification of 'errorInfo' with value 'Success' in response json failed expected [Success] but found [Surplus Location Category Doesnot exist.]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.CreateSurpluslocation(Surplus.java:66)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- CreateSurpluslocation -->
  <testcase name="createdepartmenttype" time="2.791" classname="com.schemaxtech.testAutomationFrameworkTest.DepartmentType">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.DepartmentType.createdepartmenttype(DepartmentType.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- createdepartmenttype -->
  <testcase name="CreateSurpluslocation" time="0.086" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;errorInfo&amp;apos; with value &amp;apos;Surplus Location Code already exist.&amp;apos; in response json failed expected [Surplus Location Code already exist.] but found [Surplus Location Category Doesnot exist.]">
      <![CDATA[java.lang.AssertionError: Verification of 'errorInfo' with value 'Surplus Location Code already exist.' in response json failed expected [Surplus Location Code already exist.] but found [Surplus Location Category Doesnot exist.]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.CreateSurpluslocation(Surplus.java:66)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- CreateSurpluslocation -->
  <testcase name="createdepartmenttype" time="0.098" classname="com.schemaxtech.testAutomationFrameworkTest.DepartmentType">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.DepartmentType.createdepartmenttype(DepartmentType.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- createdepartmenttype -->
  <testcase name="CreateSurpluslocation" time="0.048" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus"/>
  <testcase name="CreateSurpluslocation" time="0.097" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus"/>
  <testcase name="createPlant" time="0.136" classname="com.schemaxtech.testAutomationFrameworkTest.Plant">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Plant.createPlant(Plant.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createPlant -->
  <testcase name="UpdateSurpluslocation" time="0.12" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;errorInfo&amp;apos; with value &amp;apos;Surplus Location Updated Sucessfully&amp;apos; in response json failed expected [Surplus Location Updated Sucessfully] but found [Surplus Location Code already exist.]">
      <![CDATA[java.lang.AssertionError: Verification of 'errorInfo' with value 'Surplus Location Updated Sucessfully' in response json failed expected [Surplus Location Updated Sucessfully] but found [Surplus Location Code already exist.]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.UpdateSurpluslocation(Surplus.java:76)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- UpdateSurpluslocation -->
  <testcase name="CreateSection" time="0.111" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Section.CreateSection(Section.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- CreateSection -->
  <testcase name="createWorkstation" time="0.195" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="UpdateSurpluslocation" time="0.076" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;errorInfo&amp;apos; with value &amp;apos;Surplus Location Updated Sucessfully&amp;apos; in response json failed expected [Surplus Location Updated Sucessfully] but found [Surplus Location Code already exist.]">
      <![CDATA[java.lang.AssertionError: Verification of 'errorInfo' with value 'Surplus Location Updated Sucessfully' in response json failed expected [Surplus Location Updated Sucessfully] but found [Surplus Location Code already exist.]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.UpdateSurpluslocation(Surplus.java:76)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- UpdateSurpluslocation -->
  <testcase name="createWorkstationT" time="0.189" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;workstation_type_code&amp;apos; with value &amp;apos;planning309&amp;apos; in response json failed expected [planning309] but found [PLANNING309]">
      <![CDATA[java.lang.AssertionError: Verification of 'workstation_type_code' with value 'planning309' in response json failed expected [planning309] but found [PLANNING309]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.WorkstationT.createWorkstationT(WorkstationT.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstationT -->
  <testcase name="CreateSection" time="0.097" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Section.CreateSection(Section.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- CreateSection -->
  <testcase name="createWorkstation" time="0.15" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="UpdateSurpluslocation" time="0.234" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus"/>
  <testcase name="CreateSection" time="0.102" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Section.CreateSection(Section.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- CreateSection -->
  <testcase name="createPlant" time="0.354" classname="com.schemaxtech.testAutomationFrameworkTest.Plant"/>
  <testcase name="createWorkstation" time="0.099" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="createWorkstationT" time="0.226" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.WorkstationT.createWorkstationT(WorkstationT.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstationT -->
  <testcase name="UpdateSurpluslocation" time="0.119" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus"/>
  <testcase name="createCluster" time="0.117" classname="com.schemaxtech.testAutomationFrameworkTest.Cluster">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Cluster.createCluster(Cluster.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createCluster -->
  <testcase name="createWorkstation" time="0.119" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="createWorkstationT" time="0.172" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;workstation_type_code&amp;apos; with value &amp;apos;planning1309&amp;apos; in response json failed expected [planning1309] but found [PLANNING1309]">
      <![CDATA[java.lang.AssertionError: Verification of 'workstation_type_code' with value 'planning1309' in response json failed expected [planning1309] but found [PLANNING1309]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.WorkstationT.createWorkstationT(WorkstationT.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstationT -->
  <testcase name="createCluster" time="0.105" classname="com.schemaxtech.testAutomationFrameworkTest.Cluster"/>
  <testcase name="createWorkstation" time="0.114" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="CreateDepartment" time="0.171" classname="com.schemaxtech.testAutomationFrameworkTest.Department">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Department.CreateDepartment(Department.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- CreateDepartment -->
  <testcase name="postSampleTest" time="0.284" classname="com.schemaxtech.testAutomationFrameworkTest.Company">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;company_code&amp;apos; with value &amp;apos;comp309&amp;apos; in response json failed expected [comp309] but found [COMP309]">
      <![CDATA[java.lang.AssertionError: Verification of 'company_code' with value 'comp309' in response json failed expected [comp309] but found [COMP309]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Company.postSampleTest(Company.java:81)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- postSampleTest -->
  <testcase name="createWorkstation" time="0.068" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="CreateDepartment" time="0.119" classname="com.schemaxtech.testAutomationFrameworkTest.Department">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Department.CreateDepartment(Department.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- CreateDepartment -->
  <testcase name="postSampleTest" time="0.136" classname="com.schemaxtech.testAutomationFrameworkTest.Company">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Company.postSampleTest(Company.java:81)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- postSampleTest -->
  <testcase name="createWorkstation" time="0.134" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="createRe" time="0.241" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;reason_code&amp;apos; with value &amp;apos;Testr309&amp;apos; in response json failed expected [Testr309] but found [TESTR309]">
      <![CDATA[java.lang.AssertionError: Verification of 'reason_code' with value 'Testr309' in response json failed expected [Testr309] but found [TESTR309]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Reasons.createRe(Reasons.java:69)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- createRe -->
  <testcase name="CreateDepartment" time="0.089" classname="com.schemaxtech.testAutomationFrameworkTest.Department"/>
  <testcase name="CreatePO" time="0.273" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="createWorkstation" time="0.174" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="CreateDepartment" time="0.137" classname="com.schemaxtech.testAutomationFrameworkTest.Department"/>
  <testcase name="createRe" time="0.186" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons"/>
  <testcase name="createWorkstation" time="0.088" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.createWorkstation(Workstation.java:67)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createWorkstation -->
  <testcase name="createRe" time="0.102" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons"/>
  <testcase name="GetSection" time="0.063" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Section.GetSection(Section.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- GetSection -->
  <testcase name="updateWorkstation" time="0.102" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.updateWorkstation(Workstation.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateWorkstation -->
  <testcase name="updateCompany" time="0.085" classname="com.schemaxtech.testAutomationFrameworkTest.Company">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Company.updateCompany(Company.java:114)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- updateCompany -->
  <testcase name="updateWorkstationT" time="0.136" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT"/>
  <testcase name="getSurpluslocation" time="0.138" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.getSurpluslocation(Surplus.java:86)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- getSurpluslocation -->
  <testcase name="GetSection" time="0.086" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Section.GetSection(Section.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- GetSection -->
  <testcase name="updateWorkstation" time="0.098" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation"/>
  <testcase name="updateWorkstationT" time="0.12" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT"/>
  <testcase name="getSurpluslocation" time="0.123" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus"/>
  <testcase name="updateCompany" time="0.166" classname="com.schemaxtech.testAutomationFrameworkTest.Company"/>
  <testcase name="updateCluster" time="0.126" classname="com.schemaxtech.testAutomationFrameworkTest.Cluster">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Cluster.updateCluster(Cluster.java:80)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- updateCluster -->
  <testcase name="updateWorkstation" time="0.107" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.updateWorkstation(Workstation.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateWorkstation -->
  <testcase name="GetDepartment" time="0.118" classname="com.schemaxtech.testAutomationFrameworkTest.Department">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Department.GetDepartment(Department.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- GetDepartment -->
  <testcase name="updateCluster" time="0.077" classname="com.schemaxtech.testAutomationFrameworkTest.Cluster"/>
  <testcase name="updateRe" time="0.125" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Reasons.updateRe(Reasons.java:79)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- updateRe -->
  <testcase name="updateWorkstation" time="0.102" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.updateWorkstation(Workstation.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateWorkstation -->
  <testcase name="getSurpluslocation" time="0.158" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.getSurpluslocation(Surplus.java:86)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- getSurpluslocation -->
  <testcase name="GetDepartment" time="0.08" classname="com.schemaxtech.testAutomationFrameworkTest.Department">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Department.GetDepartment(Department.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- GetDepartment -->
  <testcase name="updateWorkstation" time="0.095" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation"/>
  <testcase name="updateRe" time="0.128" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons"/>
  <testcase name="getPo" time="0.153" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="updateWorkstation" time="0.118" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation"/>
  <testcase name="updateRe" time="0.104" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons"/>
  <testcase name="GetDepartment" time="0.186" classname="com.schemaxtech.testAutomationFrameworkTest.Department">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Department.GetDepartment(Department.java:78)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- GetDepartment -->
  <testcase name="updateWorkstation" time="0.097" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.updateWorkstation(Workstation.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateWorkstation -->
  <testcase name="updateWorkstation" time="0.087" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation"/>
  <testcase name="updateWorkstation" time="0.101" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.updateWorkstation(Workstation.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateWorkstation -->
  <testcase name="UpdateSection" time="0.0" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <skipped/>
  </testcase> <!-- UpdateSection -->
  <testcase name="deacPlant" time="0.086" classname="com.schemaxtech.testAutomationFrameworkTest.Plant">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Plant.deacPlant(Plant.java:80)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- deacPlant -->
  <testcase name="getWorkstationByWorkstaioncode" time="0.102" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.getWorkstationByWorkstaioncode(Workstation.java:77)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- getWorkstationByWorkstaioncode -->
  <testcase name="deactivateSurpluslocation" time="0.096" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;errorInfo&amp;apos; with value &amp;apos;successfully deactivated.&amp;apos; in response json failed expected [successfully deactivated.] but found [surplusLocation not found.]">
      <![CDATA[java.lang.AssertionError: Verification of 'errorInfo' with value 'successfully deactivated.' in response json failed expected [successfully deactivated.] but found [surplusLocation not found.]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Surplus.deactivateSurpluslocation(Surplus.java:96)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- deactivateSurpluslocation -->
  <testcase name="deaWorkstationT" time="0.106" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT"/>
  <testcase name="deaCompany" time="0.093" classname="com.schemaxtech.testAutomationFrameworkTest.Company"/>
  <testcase name="deacPlant" time="0.077" classname="com.schemaxtech.testAutomationFrameworkTest.Plant"/>
  <testcase name="deactivateSurpluslocation" time="0.16" classname="com.schemaxtech.testAutomationFrameworkTest.Surplus"/>
  <testcase name="UpdateDepartment" time="0.0" classname="com.schemaxtech.testAutomationFrameworkTest.Department">
    <skipped/>
  </testcase> <!-- UpdateDepartment -->
  <testcase name="deacCluster" time="0.092" classname="com.schemaxtech.testAutomationFrameworkTest.Cluster">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Cluster.deacCluster(Cluster.java:91)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- deacCluster -->
  <testcase name="deaCompany" time="0.171" classname="com.schemaxtech.testAutomationFrameworkTest.Company">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Company.deaCompany(Company.java:103)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- deaCompany -->
  <testcase name="getWorkstationByWorkstaioncode" time="0.195" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation"/>
  <testcase name="deaWorkstationT" time="0.191" classname="com.schemaxtech.testAutomationFrameworkTest.WorkstationT"/>
  <testcase name="deacCluster" time="0.107" classname="com.schemaxtech.testAutomationFrameworkTest.Cluster"/>
  <testcase name="deactRe" time="0.12" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Reasons.deactRe(Reasons.java:89)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- deactRe -->
  <testcase name="deactRe" time="0.055" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons"/>
  <testcase name="DeactivateSection" time="0.06" classname="com.schemaxtech.testAutomationFrameworkTest.Section">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Section.DeactivateSection(Section.java:100)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- DeactivateSection -->
  <testcase name="getRe" time="0.082" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Reasons.getRe(Reasons.java:99)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- getRe -->
  <testcase name="DeactivateDepartment" time="0.097" classname="com.schemaxtech.testAutomationFrameworkTest.Department"/>
  <testcase name="deActivateWorkstation" time="0.081" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.deActivateWorkstation(Workstation.java:98)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- deActivateWorkstation -->
  <testcase name="getCompany" time="0.109" classname="com.schemaxtech.testAutomationFrameworkTest.Company"/>
  <testcase name="getRe" time="0.09" classname="com.schemaxtech.testAutomationFrameworkTest.Reasons"/>
  <testcase name="DeactivateSection" time="0.15" classname="com.schemaxtech.testAutomationFrameworkTest.Section"/>
  <testcase name="getCompany" time="0.113" classname="com.schemaxtech.testAutomationFrameworkTest.Company">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Company.getCompany(Company.java:92)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- getCompany -->
  <testcase name="DeactivateDepartment" time="0.149" classname="com.schemaxtech.testAutomationFrameworkTest.Department"/>
  <testcase name="deActivateWorkstation" time="0.142" classname="com.schemaxtech.testAutomationFrameworkTest.Workstation">
    <failure type="java.lang.Exception" message="Given value not found in response json">
      <![CDATA[java.lang.Exception: Given value not found in response json
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:356)
at com.schemaxtech.testAutomationFrameworkTest.Workstation.deActivateWorkstation(Workstation.java:98)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- deActivateWorkstation -->
  <testcase name="productionOrderCreation" time="0.216" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="updateProductionOrder" time="0.06" classname="com.schemaxtech.testAutomationFrameworkTest.PPS">
    <failure type="java.lang.AssertionError" message="Verification of &amp;apos;status&amp;apos; with value &amp;apos;true&amp;apos; in response json failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verification of 'status' with value 'true' in response json failed expected [true] but found [false]
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:345)
at com.schemaxtech.testAutomationFrameworkTest.PPS.updateProductionOrder(PPS.java:353)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- updateProductionOrder -->
  <testcase name="getMasterAndSubPoSizeWiseQuantitiesByMasterPo" time="0.056" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="getMasterPoDetails" time="0.13" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="getsubPoDetailsWithMasterPo" time="0.083" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="getMasterPoStyleColorsDetails" time="0.094" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
  <testcase name="createSubProductionOrder" time="0.166" classname="com.schemaxtech.testAutomationFrameworkTest.PPS"/>
</testsuite> <!-- Test -->
